### ISSUE TITLE
try converting all lstm-metab math ops to tf math ops

### ISSUE THREAD

**Issue by jsadler2** [USGS-R/drb-do-ml#100]
There are quite a few math operations that we use in lstm-metab model. I'm not sure, but this could be one reason that the model isn't training. It's possible that b/c these aren't tensorflow operations, the gradients are not being properly propagated through them. Which would explain why the LSTM-metab model isn't training. This is something that Andy and Xiaowei brought up. I'd like to try converting everything to tensorflow operations to see what happens.

**Comment by jsadler2** [comment]
I want to change things incrementally.
1. change `**` to `tf.math.pow`
2. change `*` and `/` to `tf.math.multiply` and `tf.math.divide`
3. change `+` and `-` to `tf.math.add` and `tf.math.minus`

**Comment by jsadler2** [comment]
1. Changing `**` to `tf.math.pow` didn't seem to make much of a difference

![image](https://user-images.githubusercontent.com/6943441/168173024-16ad4ee9-2017-461d-b905-495e528bc930.png)

**Comment by jsadler2** [comment]
Even after changing all of the math operations to `tf.math` functions, the model still does not train:
![image](https://user-images.githubusercontent.com/6943441/168897090-1a9ef230-5d05-4d56-8764-aff40ccc0d02.png)
